DigitalLearningSolutions.Web/Views/Home/LearningContent.cshtml

```diff
@@ -0,0 +1,73 @@
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.Home
+@model LandingPageViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/learningContent.css")"/>
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">
+
+    <h2>Learning Content</h2>

```

Comment:
I think we might want these to be h1s as per https://service-manual.nhs.uk/design-system/patterns/mini-hub
Also, could we add the font style explicitly here - https://service-manual.nhs.uk/design-system/styles/typography#headings

Reply:
Good point. I'm also thinking we should add a sub-heading to the Products and LearningContent sections so it's clear the user's in the landing page. Not sure what it should be though. "Welcome"? Feels a bit weak...

Reply:
If unsure, you could check in the external HEE channel as they're in charge of the content anyway. :)